Examples/STM32/uart_cli/README.md

```diff
@@ -54,6 +54,19 @@ Alternatively, use an IDE to compile and flash the project into the NUCLEO board
 2. Select the bus configuration (**SPI or SDIO**). Make sure it matches the switch position on the WF(M)200 expansion board.
 3. Build the project.
 4. Flash the project to the NUCLEO-F429ZI board.
+5. Please be aware that:
+-  if you are using newest versions of IAR Embedded Workbench, the output messages will be displayed on the **Terminal I/O window** of the IAR Embedded Workbench by default when you run the project through the **debug mode**. 
+- Therefore, if you want to **redirect printf to UART** to display the output messages on your serial terminal like **Tera Term** or **Putty**, you need to override the **default function __write** of IAR Embedded Workbench.
+- One way to do that is using our customized **printf.c file**. You can find the source code in the Examples\STM32\shared\Middlewares\Third_Party\printf directory. Once you have the source code, you only need to add the file into your project.
```

Comment:
For example:
One way to do that is using a customized ... file, (embed a hyper link to the folder). Add the file to (where) of the project